.travis.yml

```diff
@@ -9,6 +9,7 @@ branches:
   except:
     - piptools-ignore-patch
 install:
+  - "python -m pip install pip --upgrade"
```

Comment:
By the way, by comparing [build 57.2](https://travis-ci.org/jgonggrijp/pip-review/jobs/194283839) and [build 58.2](https://travis-ci.org/jgonggrijp/pip-review/jobs/194284781), you can also see that adding the update line in `.travis` did not make a difference. Something else is going on, which you were trying to fix.